res/css/structures/_UploadBar.pcss

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 
 .mx_UploadBar {
     padding-left: 65px; /* line up with the shield area in the composer */
+    padding-top: 2px;
```

Comment:
This creates a weird 7px spacing that we don't use anywhere else.
I think it would be cleaner to replace `.mx_UploadBar_filename` `margin-top` with this padding. That is - remove the `margin-top` from `.mx_UploadBar_filename`, and set this `.mx_UploadBar` `padding-top` to 5px.